src/ServerConfigParams.ts

```diff
@@ -71,7 +71,12 @@ export interface ServerConfigParams {
      * @note the minus sign is omitted if used
      * @note fractions exceeding the number of decimals of that of the native currency will be rounded up
      */
-    feePercentage: string;
+    feePercentage?: string;
+
+    /**
+     * Destinations list that will make the relay worker pay for the transaction if @disableSponsoredTx is false
```

Comment:
Not sure if my proposal is better, but that description isn't very clear to me 😅
```suggestion
     * List of destination contract addresses for which the transactions will be [sponsored](https://developers.rsk.co/rif/relay/architecture/#glossary) if @disableSponsoredTx is false.
```

Reply:
updated